app/assets/src/components/BulkUploadImport.jsx

```diff
@@ -65,14 +74,6 @@ class BulkUploadImport extends React.Component {
   componentDidMount() {
     $("body").addClass("background-cover");
     this.initializeSelectTag();
-    $(ReactDOM.findDOMNode(this.refs.projectSelect)).on(
```

Comment:
These seem not needed anymore, since those elements have `onChange` event handler directly?